javascript/MaterialXTest/element.spec.js

```diff
@@ -164,3 +164,148 @@ describe('Element', () =>
         });
     });
 });
+
+describe('Equivalence', () =>
+{
+    let mx, doc, doc2, inputMap, inputMap2, floatInputs;
+
+    before(async () => {
+        mx = await Module();
+
+        doc = mx.createDocument();
+        inputMap = new Map([
+            ["color3", "  1.0,   +2.0,  3.0   "],
```

Comment:
Since these JavaScript bindings are simple wrappers around the underlying C++ methods, perhaps we don't need these additional unit tests in JavaScript, which mirror the same setup as our C++ tests?

My sense is that the new bindings in `JsElement.cpp` are sufficient here, and we can let our existing C++ unit tests do the heavy lifting!

Reply:
I've simplified it to just test where there are C++ logic for wrapper for `isEquivalent()`

Reply:
Maybe I'm missing something important here, but I was imagining that this entire setup (e.g. the extensive `inputMap` and `inputMap2`, the testing of various difference categories, and so forth) could be left in C++, and that our JavaScript test would be as simple as verifying that the JavaScript binding works at all -- e.g. that the `isEquivalent` method can be called without triggering a JavaScript error message.

Reply:
You're correct. I forgot to remove the complex setup.